packages/frontend/src/views/CalendarView/CalendarView.test.tsx

```diff
@@ -0,0 +1,3 @@
+import '@testing-library/jest-dom';
+
+test('first test', () => {});
```

Comment:
Can commit this file when there are actual tests xD

Reply:
WOOPS yes will remove